lib/fog/core/model.rb

```diff
@@ -20,6 +21,10 @@ def initialize(new_attributes = {})
       merge_attributes(attribs)
     end
 
+    def cache
+      Fog::Cache.new(self)
+    end
```

Comment:
Hmmm. Actually all methods are available through `.class` on the instance but let it be. =)
We can refactor this later. No need to block this PR because of this.